keyform/management/commands/migrate_old_data.py

```diff
@@ -0,0 +1,107 @@
+import sqlite3
+from datetime import datetime
+from django.core.management.base import BaseCommand
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.utils.timezone import make_aware, utc
+from keyform.models import *
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        with transaction.atomic():
+            c = sqlite3.connect('/tmp/keyform.db').cursor()
+
+            buildings = {}
+            for row in c.execute('select building_id, building_name from building'):
+                buildings[str(row[0])] = Building.objects.get_or_create(name=row[1])[0]
```

Comment:
Did anyone check to make sure there is no odd spelling for the same building so we do not create duplicate buildings?